mne/io/ctf/ctf.py

```diff
@@ -45,6 +45,9 @@ def read_raw_ctf(directory, system_clock='truncate', preload=False,
     verbose : bool, str, int, or None
         If not None, override default verbose level (see :func:`mne.verbose`
         and :ref:`Logging documentation <tut_logging>` for more).
+    clean_names : bool, optional
+        If True main channel names and compensation channel names will
+        be cleaned from CTF suffixes. The default is False.
```

Comment:
`verbose` should always be the last argument (by internal convention)

Reply:
@larsoner Should `verbose` be kept last in function signature and documentation?

Reply:
Yes both